test/Microsoft.DotNet.SourceBuild.Tests/assets/LicenseScanTests/LicenseExclusions.txt

```diff
@@ -185,22 +185,14 @@ src/sdk/THIRD-PARTY-NOTICES.TXT|unknown-license-reference
 #
 
 # False positive
-src/source-build-externals/src/abstractions-xunit/README.md|free-unknown
-src/source-build-externals/src/application-insights/NETCORE/ThirdPartyNotices.txt|unknown
-src/source-build-externals/src/humanizer/NuSpecs/*.nuspec*
-src/source-build-externals/src/xunit/README.md|free-unknown
-src/source-build-externals/src/xunit/src/xunit.assert/Asserts/README.md|free-unknown
-src/source-build-externals/src/xunit/tools/builder/common/README.md|free-unknown
```

Comment:
Related to https://github.com/dotnet/dotnet/pull/451, need to update the file path: `src/source-build-externals/src/repos/src`